### ISSUE TITLE
Using Normalizing Flows with Pytorch-forecasting

### ISSUE THREAD

**Issue by sege01** [jdb78/pytorch-forecasting#552]
Question:- Hello! Thank you for the wonderful work.
I am new to pytorch-forecasting. I had a question regarding the use of Normalizing Flows with pytorch-forecasting. Is it possible to implement the Normalizing Flows for Multivariate time-series with the pytorch forecasting?

**Comment by jdb78** [comment]
Yes. This should be possible in principal. See also #544. You are welcome to work on the PR for the normalizing flow. I have not for some time but I think generally speaking, there should not be too many additional pieces that require changing apart from what has been started in the PR. I might work on it in the future but, unfortunately, cannot provide you with a timeline.
